sign-up.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>SpaceWalkers</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
+  <link href="css/index.css" rel="stylesheet">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+ <!-- navigation code here -->
+  <div class="title-nav">
+    <h1>SpaceWalkers</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </nav>
+  </div>
+
+  <div class="container home">
+
+    <header class="intro">
+       <!-- image code here -->
+      <img src="img/main-header.png">
+      <h2>Lift Off</h2>
+    
+      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+    </header>
+
+    <form>
+        First name:<br>
+        <input type="text"><br>
+        Last name:<br>
```

Comment:
Take a look at W3 schools or MDN docs about the `input` HTML tag. It has alot of really cool properties you can give it like `type`, `placeholder`, `name`, etc... This will make it easier to work with. The reason you cant now is that you have floating content on lines `43, 45, 47, and  49`